### PR TITLE
Added new exmaple and removed js from static

### PR DESCRIPTION
**Pull request by tmccoy529**
Added a new example for static step lists. This stops the javascript from working on the step list row and removes unused elements in the step list. Added to html file and md file.

### CODE REVIEW

File: src/components/StepList.md [hunk 1]
```diff
@@ -75,13 +169,17 @@ The `id` of the step much match the step detail's `aria-labelledby`. In the exam
 **Step**
 
 ```html
-<li class="dg_step-list__list-section" id="registration-step-1">
+<li class="dg_step-list__list-section" id="registration-step-1"></li>
```
Comment (martypowell) [inline src/components/StepList.md]: is the `</li>` intentional here?

**Comment by martypowell** [comment]
@tmccoy529 do we know if this is still legitimate?

**Comment by tmccoy529** [comment]
@martypowell this was addressing a bug with regards to being able to tab into a static steplist which then highlights it. But I just tested this in the library and it doesnt seem to do this anymore. Id say this can be closed.

**Comment by martypowell** [comment]
closing per @tmccoy529's comments

**Pull request closed without merging**